api/tokens_test.go

```diff
@@ -139,12 +141,109 @@ func TestHandleUnexpectedError(t *testing.T) {
 		So(resp.Body.String(), ShouldResemble, errorResponseBodyExample)
 	})
 }
+func TestCognitoResponseBuild(t *testing.T) {
```

Comment:
Slightly confusing name, should it not be TestCognitoRequestBuild?

Reply:
This has been updated to the suggestion